### DISCUSSION TITLE
Hello, why the resolve dependencies from workspace projects just not working

### DISCUSSION THREAD

**Discussion by WestFarmer** [eclipse-m2e/m2e-core#1616] [Q&A]
I am running the lastest eclipse 2023-09

I have two projects A and B

A was a snapshot version, and was referenced as dependency by B

I modify the code of A, why not take effect when running B...

**resolve dependencies from workspace projects** was checked!

check the maven dependencies, A was recognized as workspace project dependency.

![图片](https://github.com/eclipse-m2e/m2e-core/assets/14016562/b1fda882-6bf3-4a18-be58-3cf54c5e6f92)

**Comment by mickaelistria** [comment]
Can you please open a bug report, with detailed steps to reproduce the issue from a fresh installation of Eclipse IDE with latest release or snapshot of m2e?

**Comment by WestFarmer** [comment]
@mickaelistria 
my bad, this is a false report.
